observers/run-observations.sh

```diff
@@ -75,7 +75,10 @@ run_recipe_if_needed() {
   
   if [ -f "$adapted_recipe_path" ]; then
     echo "$(date): Found adapted recipe: $adapted_recipe_path"
-    recipe_to_run="$adapted_recipe_path"
+    # Naive validation for required fields
+    if grep -q "title:" "$adapted_recipe_path" && grep -q "description:" "$adapted_recipe_path"; then
+      recipe_to_run="$adapted_recipe_path"
+    fi
```

Comment:
If the adapted recipe can't be run, we should use the original recipe instead. My `recipe-background-tasks.yaml` was failing because `title` was removed. I was considering creating a new `fix-recipe.yaml` action, but starting simple.